python/pyspark/sql/functions/builtin.py

```diff
@@ -13090,6 +13090,108 @@ def substr(
         return _invoke_function_over_columns("substr", str, pos)
 
 
+@_try_remote_functions
+def try_parse_url(
+    url: "ColumnOrName", partToExtract: "ColumnOrName", key: Optional["ColumnOrName"] = None
+) -> Column:
+    """
+    URL function: Extracts a specified part from a URL. If a key is provided,
```

Comment:
Could you update this description to give the similar message as other try functions?